api/handlers/service_binding_test.go

```diff
@@ -50,6 +52,12 @@ var _ = Describe("ServiceBinding", func() {
 			Type:      korifiv1alpha1.UserProvidedType,
 		}, nil)
 
+		serviceBindingRepo.GetServiceBindingDetailsReturns(repositories.ServiceBindingDetailsRecord{
```

Comment:
Sertting up the serviceBindingRepo to return the binding details only makes sense to `Describe("GET /v3/service_credential_bindings/{guid}/details"`, the rest of the test does not need it. Could you move that in that `Describe` `BeforeEach`?